src/api.ts

```diff
@@ -738,15 +738,16 @@ export class APIGatewayVTL extends VTL {
       } else if (isReferenceExpr(expr.expr) || isThisExpr(expr.expr)) {
         const ref = expr.expr.ref();
         if (ref === Number) {
-          return this.exprToJson(expr.args[0]);
+          // Number() = 0
+          return expr.args[0] ? this.exprToJson(expr.args[0]) : "0";
```

Comment:
why `"0"` instead of `0`?

Reply:
its returning a string? Above, returning a numeric literal looks like `number.toString(10)`